src/Core/Model.php

```diff
@@ -461,6 +461,16 @@ public static function getFields() {
     return self::getInstance()->_getFields();
   }
 
+  private function _getCapabilities() {
+    if(!isset($this->caps))
+      return false;
+    return $this->caps;
+  }
+
+  public static function getCapabilities() {
+    return self::getInstance()->_getCaps();
```

Comment:
@caiovncius Tests are failing here, please change to `return self::getInstance()-> _getCapabilities()`